book/website/community-handbook/coworking/coworking-dropins.md

```diff
@@ -0,0 +1,120 @@
+3. Online github repo: share information about the session, including [the schedule](https://github.com/alan-turing-institute/infrastructure-drop-ins/wiki/Schedule) and [how to sign up](https://github.com/alan-turing-institute/infrastructure-drop-ins)
+4. Engagement and impact tracking resource: Sheet with information on the topics raised, next steps, and the overall impact of the session
+
+Sessions may or may not be recorded.
+If they are recorded, they can serve as a resource for attendees to come back to.
+It they aren't recorded, this may enable attendees to speak more freely and openly during the session.
+
+Breakout rooms may be created if there are multiple sessions happening at the same time.
+In that case, one host will stay in the main room to welcome new attendees and other hosts can meet with attendees in different breakout rooms.
+
+## Format of the session
+
+The following is an example format of a drop-in session.
+
+| Duration | Activity |
+| ---- | -------- |
+| 5 mins | 👋 Welcome attendees and assign slot |
+| 30 mins | :speech_balloon: Slot 1 |
+| 30 mins | :speech_balloon: Slot 2 |
+| 30 mins | :speech_balloon: Slot 3 |
+| 30 mins | :speech_balloon: Slot 4 |
+| 5 mins | 👋 Close session |
+
+Each slot may run longer or shorter than 30 minutes, and multiple slots may run in parallel through breakout rooms.
+
+## Chairing drop-in sessions
+
+### Before the Session
+
+- Session hosts can double check the schedule to make sure the correct information is documented for that session
+    - Update host names and affiliations
+    - Correct Zoom link and physical location of the session
+    - Share session on Slack and other channels inviting attendees to the event
+- Session hosts can coordinate amongst each other to schedule and assign slots during the session
+
+### During the Session
+
+- Make sure there is someone monitoring the main Zoom room and physical space to welcome new attendees
+- It is especially important that someone is available before each slot to guide attendees to a breakout room, if those are being used 
+
+### After the Session
+
+Record notes for next steps and follow-ups in a shared document so that teammates can pick up conversations and collaborations in future sessions.
+
+## Case Study: Turing Infrastructure Drop-In Sessions
+
+At the Turing Institute, there are a few different variations of drop-in sessions including the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) (online and in person), the Turing Research Ethics (TREx) Virtual Drop in Session (online only), the Turing Skills Drop in Session (online and in person), and the Data Protection virtual drop in (online only).
+
+This section will highlight the Turing Infrastructure Drop-In Sessions as a case study to illustrate how a drop-in session can run in practice.
+
+### Attending a drop-in session
```

Comment:
Starting from here it becomes less informative compared to before - the information you share here is already available above. So I would agree with Arielle that a case study format would make more sense.

Reply:
Thanks for the feedback, @EstherPlomp! Originally the content was interlinked and I think it is much better split up like this. Do you think it's worth including the case study as an example (as written), or is it too repetitive? The primary value I see is sharing the public github link for the Turing sessions which could help "ground" the chapter.

Another approach could be to change the "Case Study" section to a "List of Examples" section.

Reply:
I do like it as a case study so I would still include it! But then indeed more as a case study or List of examples to clarify that it may not contain super new information!